src/nile/cli.py

```diff
@@ -88,12 +87,23 @@ def deploy(artifact, arguments, network, alias):
 
 
 @cli.command()
-@click.argument("artifact", nargs=1)
+@click.argument("signer", nargs=1)
+@click.argument("contract_name", nargs=1)
+@click.option("--max_fee", nargs=1)
 @network_option
 @click.option("--alias")
-def declare(artifact, network, alias):
+@click.option("--directory")
```

Comment:
```suggestion
@click.option("--contracts_directory")
```

Reply:
chose an alternative version. i prefer keeping the cli arg name short, what do you think?

Reply:
Easier to type, totally fine by me